extensions/sql-migration/src/constants/strings.ts

```diff
@@ -433,3 +435,16 @@ export function WARNINGS_COUNT(totalCount: number): string {
 export const AUTHENTICATION_TYPE = localize('sql.migration.authentication.type', "Authentication Type");
 export const SQL_LOGIN = localize('sql.migration.sql.login', "SQL Login");
 export const WINDOWS_AUTHENTICATION = localize('sql.migration.windows.auth', "Windows Authentication");
+
+export const FEEDBACK_DIALOG_SUBMIT_BUTTON = localize('sql.migration.feedback.submit.button', "Submit");
+export const FEEDBACK_DIALOG_CANCEL_BUTTON = localize('sql.migration.feedback.cancel.button', "Cancel");
+export const FEEDBACK_DIALOG_TITLE = localize('sql.migration.feedback.title', "Submit Feedback");
+export const FEEDBACK_DIALOG_HEADING = localize('sql.migration.feedback.heading', "Overall, how satisfied or dissatisfied are you with Azure SQL Migration experience?");
+export const FEEDBACK_DIALOG_PLACEHOLDER = localize('sql.migration.feedback.placeholder', "We appreciate your feedback. How can we improve? (optional).");
```

Comment:
I would include information about the max allowed length in the placeholder.

Reply:
I just realized the input component doesn't have a max length property, I will add one.

Reply:
that was the issue I found.  I also noticed the 'place holder text' would not wrap as  a style is being applied while 'empty'.. resulting in the placeholder text running offscreen

Reply:
please open an issue for it. thanks

Reply:
@brian-harris you don't have to open an issue now, I have a fix for the 2 issues mentioned here: https://github.com/microsoft/azuredatastudio/pull/15587

Reply:
Thank you @alanrenmsft.

Reply:
My PR has been merged, now you can set the max length to 500 and also use a more descriptive place holder.